packages/tests/tests/utils.spec.ts

```diff
@@ -121,148 +116,3 @@ describe("Util: toAsyncIterator: Filter", () => {
     expect(result.done).to.eq(true);
   });
 });
-
-const TestCodec = "test/1";
-
-describe("selectPeerForProtocol", () => {
```

Comment:
do all these tests need to be added elsewhere?

Reply:
right -- refactored tests and added here: b0fd0e36e8da8ad1c31e098106418505711af7d0